apps/docs/content/guides/realtime/broadcast.mdx

```diff
@@ -637,3 +637,130 @@ You can also send a Broadcast message by making an HTTP request to Realtime serv
   Unsupported in Python yet.
   </TabPanel>
 </Tabs>
+
+
+## Trigger broadcast messages from your database
+<Admonition type="caution">
+This feature is currently in Private Alpha. To use Broadcast Changes request them via Support Ticket.
+</Admonition>
+
+<Admonition type="caution">
+This feature is currently in Private Alpha. It's subject to changes both on implementation and usability
+</Admonition>
+
+### How it works
+
+Broadcast Changes allows you to trigger messages from your database. To achieve it we're directly reading your WAL file using a publication against the `realtime.messages` table so whenever a new insert happens we are able to broadcast it to your users.
+
+We are using partitioned tables per day which allows us to delete your previous images in a performant way by dropping the physical tables of this partitioned table. Currently we are dropping tables older than 3 days old.
+
+The broadcast itself uses the same approach as our [Broadcast](https://supabase.com/docs/guides/realtime/broadcast?queryGroups=language&language=js) feature which utilizes WebSockets to send JSON packages. We've also set [Realtime Authorization](https://supabase.com/docs/guides/realtime/authorization?queryGroups=language&language=dart)  as a requirement from the start to ensure connected users comply to your Authorization rules and avoid information leakage.
+
+To help you with this we provide you two functions:
+
+- `realtime.send` that will do the insert into `realtime.messages`
+- `realtime.broadcast_changes` that will use `realtime.send` but help you setup the fields required to emit changes to your clients easily.
+
+### Broadcasting a message from your database
+
+We wanted to provide you as much flexibility as possible so we have `realtime.send` that will enable you to broadcast messages from your database without a specific format:
```

Comment:
The `realtime.send` function provides the most flexibility by allowing you to broadcast messages from your database without a specific format. This allows you to use database broadcast for messages that aren't necessarily tied to the shape of a Postgres row change.